ta-generator/golden/sast-snyk-check/ta.yaml

```
@@ -91,7 +96,7 @@ spec:
 
         SNYK_EXIT_CODE=0
         SOURCE_CODE_DIR=/var/workdir/source
-        snyk code test $ARGS $SOURCE_CODE_DIR --sarif-file-output=sast_snyk_check_out.json 1>&2>> stdout.txt || SNYK_EXIT_CODE=$?
+        snyk code test "$ARGS" "$SOURCE_CODE_DIR" --max-depth=1 --sarif-file-output=sast_snyk_check_out.json 1>&2>> stdout.txt || SNYK_EXIT_CODE=$?
```

I don't think you want to quote ARGS

```suggestion
        # shellcheck disable=SC2086
        # We do want to expand ARGS (it can be multiple CLI flags, not just one)
        snyk code test $ARGS "$SOURCE_CODE_DIR" --max-depth=1 --sarif-file-output=sast_snyk_check_out.json 1>&2>> stdout.txt || SNYK_EXIT_CODE=$?
```

(a better solution could be to make ARGS an array rather than a single string, but that would be another breaking change)